systemtest/jpsreport_tests/method_A/test3/03_method_A.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python3
+#---------
+# one pedestrian created at frame 9
+# pedestrian starts in position (0, 1) and moves forward
+# in a straight corridor with a speed of 1 m/s
+#
+# arrival times should be:
```

Comment:
Are you comparing two measuring points? The old test compared two time series. What does the result tell us?

Reply:
Solved due to various explanations in this PR